rest-api/config/base_config.json

```diff
@@ -24,6 +24,9 @@
     "1PS08",
     "1SS08",
     "1UR10",
+    "1CFD9",
+    "1PXR2",
+    "1UR90",
```

Comment:
Can we confirm that 1UR90 (and the other tests) don't belong in dna_sample_test_codes?

Reply:
it is confirmed